### PR TITLE
get nameplate

### PR DESCRIPTION
**Pull request by TheBunnyMan123**
this has been thoroughly stress tested in my avatar

### CODE REVIEW

File: common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java [hunk 1]
```diff
@@ -270,6 +273,34 @@ public void loadAvatar(@LuaNotNil String playerUUID) {
...
+      }
+
+      NameplateAPI plate = avatar.luaRuntime.nameplate;
+
+      String name = avatar.entityName;
+      String chat = ObjectUtils.firstNonNull(plate.CHAT.getText(), name, avatarUUID);
+      String entity = ObjectUtils.firstNonNull(plate.ENTITY.getText(), name, avatarUUID);
+      String list = ObjectUtils.firstNonNull(plate.LIST.getText(), name, avatarUUID);
+
+      return new String[]{chat, entity, list};
```
Comment (PoolloverNathan) [inline common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java:301]: personally I think this should be a named table

**Review by PoolloverNathan — CHANGES_REQUESTED** [review]
(feel free to dismiss if you disagree)

File: common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java (hunk 1 above)
Comment (TheBunnyMan123) [reply common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java:301]: that is how you return multiple things on the lua side

Comment (PoolloverNathan) [reply common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java:301]: and you're only telling me this now?? i thought you needed to construct a `Varargs`

**Review by PoolloverNathan — APPROVED** [review]
ah, carry on

**Pull request merged**